xml/hwbp_registers.xml

```diff
@@ -220,6 +220,19 @@
         hold all valid virtual addresses but it need not be capable of holding
         other values.
 
+        If the A extension is supported, then trigger behavior is as follows
+        for the load and store bits:
+        \begin{tightlist}
```

Comment:
I'm not sure what exactly tightlist does, but I'm guessing it's like steps which I made for this document. So write:
`\begin{steps}{If the A extension is supported, then trigger behavior is as follows for the load and store bits:}`
which also eliminates the empty line between the text and the first list entry.

Reply:
I really think that this should be an unordered list and I picked tightlist over itemize to reduce empty space.

Reply:
I reduced the empty space.